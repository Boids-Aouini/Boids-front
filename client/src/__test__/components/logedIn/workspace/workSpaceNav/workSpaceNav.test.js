import React from 'react';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store';
import { shallow } from 'enzyme';
import WorkSpaceNav from '../../../../../components/logedIn/workSpace/workSpaceNav/workSpaceNav';

let mockStore = configureStore();
let initState = {};
let store = mockStore(initState);

describe('<WorkSpaceNav />', () => {
    describe('render()', () => {
        test('render without crashing', () => {
            shallow(<WorkSpaceNav store={store} />);
        })
        test('render component', () => {
            let wrapper = shallow(<WorkSpaceNav store={store} />);
            let component = wrapper.dive();
            expect(toJson(component)).toMatchSnapshot();
        })
    })
})