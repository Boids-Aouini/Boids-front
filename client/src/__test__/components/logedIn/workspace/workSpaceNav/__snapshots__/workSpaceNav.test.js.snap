// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WorkSpaceNav /> render() render component 1`] = `
<WorkSpaceNav
  makeChannel={[Function]}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
/>
`;
